src/components/InlineEdit/InlineEdit.tsx

```diff
@@ -8,15 +8,16 @@ import {
   Ref,
 } from "react";
 import tw, { styled } from "twin.macro";
+import "styled-components/macro";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { PencilIcon } from "@heroicons/react/solid";
```

Comment:
what is the difference between these two packages?

Reply:
There might be some minor differences in the UI which we need to clarify with Darius or Jeremiah, but the Tailwind official uses Heroicons. So we started using Heroicons in the project, so I continued using the same, but in this particular component, I saw FontAwesomeIcon. not sure if we changed the library that has to be used, or we continue with both, or we don't care about it. 

FYI: I know UI styling is not our first priority but, I just added some styling to the Icon, so that it doesn't look clumpsy.

Reply:
Thanks, @RohitKandimalla for clarification. @gregory-akins do we need two different sets of icon packages?